currency_monthly_rate/README.rst

```diff
@@ -0,0 +1,74 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+=====================
+Currency Monthly Rate
+=====================
+
+This module extends the functionality of base module to support monthly average
```

Comment:
Module name can be misleading (there is already a monthly rate in Odoo) while the module actually introduces `average`.
What about `currency_monthly_average_rate` or simply `currency_average_rate`?
(or maybe I am confused with the feature...)

Reply:
Actually we firstly thought about using the 'average' term but as this module doesn't compute any average, we removed it.
The module allows to define one and only one rate per month, and from what I know it does not exist in odoo...